pkg/vadmin/remove_sc_vc.go

```diff
@@ -17,15 +17,70 @@ package vadmin
 
 import (
 	"context"
-	"fmt"
+	"errors"
 
+	"github.com/vertica/vcluster/rfc7807"
+	vops "github.com/vertica/vcluster/vclusterops"
+	"github.com/vertica/vcluster/vclusterops/vstruct"
+	vapi "github.com/vertica/vertica-kubernetes/api/v1beta1"
+	"github.com/vertica/vertica-kubernetes/pkg/net"
 	"github.com/vertica/vertica-kubernetes/pkg/vadmin/opts/removesc"
 )
 
 // RemoveSubcluster will remove the given subcluster from the vertica cluster.
 func (v *VClusterOps) RemoveSubcluster(ctx context.Context, opts ...removesc.Option) error {
 	v.Log.Info("Starting vcluster RemoveSubcluster")
+
+	// get the certs
+	certs, err := v.retrieveHTTPSCerts(ctx)
+	if err != nil {
+		return err
+	}
+
 	s := removesc.Parms{}
 	s.Make(opts...)
-	return fmt.Errorf("not implemented")
+
+	// call vcluster-ops library to remove_subcluster
+	vopts := v.genRemoveSubclusterOptions(&s, certs)
+	_, err = v.VRemoveSubcluster(&vopts)
+	if err != nil {
+		vproblem := &rfc7807.VProblem{}
+		if ok := errors.As(err, &vproblem); ok {
+			if vproblem.Type == rfc7807.SubclusterNotFound.Type {
+				// Nothing to do if the subcluster is already gone.
+				v.Log.Info("Attempted to remove a subcluster that was already gone", "subcluster", s.Subcluster)
+				return nil
+			}
+		}
+	}
+
+	return err
+}
+
+func (v *VClusterOps) genRemoveSubclusterOptions(s *removesc.Parms, certs *HTTPSCerts) vops.VRemoveScOptions {
+	opts := vops.VRemoveScOptionsFactory()
+
+	// required options
+	opts.Name = &v.VDB.Spec.DBName
+	opts.SubclusterToRemove = &s.Subcluster
+
+	*opts.HonorUserInput = true
+	opts.RawHosts = append(opts.RawHosts, s.InitiatorIP)
```

Comment:
Is `opts.RawHosts` empty prior to this? Based on the comments in the PR, it sounds like we also will remove nodes, but I believe this is handled by a prior step in the reconcile. If so, I suggest we update the PR description and make this change to clarify this:

```suggestion
	opts.RawHosts = []string{s.InitiatorIP}
```

Reply:
`remove_sc` also removes the nodes belonging to the sc to removed that's what I meant. But I will change it to just say `to remove a subcluster`